conf.d/health_alarm_notify.conf

```diff
@@ -113,6 +113,24 @@ DEFAULT_RECIPIENT_PUSHOVER=""
 
 
 ###############################################################################
+
+###############################################################################
+# pushbullet (pushbullet.com) push notification options
+
+# multiple recipients can be given like this:
+#                  "USERTOKEN1 USERTOKEN2 ..."
+
+# enable/disable sending pushover notifications
+SEND_PUSHBULLET="YES"
+
+# Login to pushover.net to get your pushover app token.
```

Comment:
it says pushover.net is a several places, instead of pushbullet...
Is app token needed for pushbullet?